config/rbac/service_account.yaml

```diff
@@ -3,3 +3,53 @@ kind: ServiceAccount
 metadata:
   name: controller-manager
   namespace: system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: ansibleee
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: ansibleee-role
+  namespace: openstack
+rules:
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - anyuid
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - post
+  - update
+  - watch
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: ansibleee-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: ansibleee-role
+subjects:
+# Applying the role to both SA (with and without prefix) to be able
+# to run the operator local
+- kind: ServiceAccount
+  name: openstack-ansibleee-operator-ansibleee
```

Comment:
If we keep this consistent with the other operators then `openstack-ansiblee-operator-openstack-ansiblee` looks consistent, though I don't have strong opinion about this.

Reply:
i reply here for all your comments: before my last two reverts I did it this way, but, I don't know, the SA sounded strange: "openstack-ansibleee-operator-openstack-ansibleee". The name sounded too long and it contained "openstack" word 2 times only because this repo is named different from the others. But i can reapply those commits, both ways are fine for me. :man_shrugging: